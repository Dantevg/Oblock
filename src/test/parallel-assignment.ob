var a, b, c
a, b := 10
a, b, c := a, nil, 30
print(a, b, c)
a, b := b, a
print(a, b, c)
