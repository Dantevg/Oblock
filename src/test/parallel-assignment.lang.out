10	nil	30
nil	10	30
